spec/unit/policy_builder/policyfile_spec.rb

```diff
@@ -330,6 +330,34 @@ def initialize_pb
 
       end
 
+      describe "#build_node" do
+
+        let(:node) do
+          node = Chef::Node.new
+          node.name(node_name)
+          node
+        end
+
+        before do
+          allow(policy_builder).to receive(:node).and_return(node)
+        end
+
+        context "when the run is successful" do
+          let(:expanded_run_list) { ["test::default@0.1.0 (9a156f6)",
+                                     "test::other@0.1.0 (ae23e31)"] }
```

Comment:
Do we know of anything that parses these strings? Seems they may likely do the wrong thing with this format